examples/morpheusvm/cmd/morpheus-cli/cmd/resolutions.go

```diff
@@ -52,8 +52,9 @@ func handleTx(tx *chain.Transaction, result *chain.Result) {
 	status := "⚠️"
 	if result.Success {
 		status = "✅"
-		switch action := tx.Action.(type) { //nolint:gocritic
-		case *actions.Transfer:
+		switch tx.Action.GetTypeID() { //nolint:gocritic
+		case (&actions.Transfer{}).GetTypeID():
+			action := tx.Action.(*actions.Transfer)
```

Comment:
Ah, I didn't think about the need to do this...probably better to just leave for now.